src/abax-client.ts

```diff
@@ -94,27 +95,14 @@ export class AbaxClient {
     return this.performRequest(apiKey => call({ input, apiKey }));
   }
 
-  /** Gets paged list of Trips. Required scopes: `abax_profile`, `open_api`, `open_api.trips`.  */
-  listTrips(input: ListTripsInput): Promise<ListTripsResponse> {
```

Comment:
Do we want a new docstring for this method? :eyes:

Reply:
It still gets a paged list of trips. The input interface explains that you can set page_size to 0 to get all trips in one page